worker/include/Channel/ChannelRequest.hpp

```diff
@@ -22,14 +22,15 @@ namespace Channel
 
 	public:
 		static absl::flat_hash_map<FBS::Request::Method, const char*> method2String;
+		thread_local static flatbuffers::FlatBufferBuilder bufferBuilder;
 
 	public:
 		ChannelRequest(Channel::ChannelSocket* channel, const FBS::Request::Request* request);
 		~ChannelRequest() = default;
 
 		flatbuffers::FlatBufferBuilder& GetBufferBuilder()
 		{
-			return this->bufferBuilder;
+			return ChannelRequest::bufferBuilder;
```

Comment:
@jmillan you have approved this PR but here we are using a thread static member instead of a instance member, so I don't understand.